app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
```

Comment:
- [ ] We want your design to closely match the given  login wireframe design. Kindly improve upon your design.


**Your login page:**
![image](https://user-images.githubusercontent.com/12745474/186496509-ebf767cb-6978-4af8-87c6-35a06b53b4c0.png)

**Wireframe login page:**
![image](https://user-images.githubusercontent.com/12745474/186496816-fbcdd0b9-c489-4399-b7d0-766edfb19b79.png)